kolibri/plugins/coach/assets/src/views/exams/exams-page/index.vue

```diff
@@ -122,7 +122,7 @@
         return ContentNodeKinds.EXAM;
       },
       sortedExams() {
-        return orderBy(this.exams, [exam => exam.title.toUpperCase()], ['asc']);
+        return reverse(this.exams);
```

Comment:
`reverse` modifies the array in-place.

You should either [copy the array first](https://stackoverflow.com/a/30610528) or use `lodash/fp/reverse`

Reply:
(also note - there's a [built-in reverse function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse) function).

Reply:
Oops, good catch!